tests/README.md

```diff
@@ -163,125 +313,22 @@ Example of a scenario:
-- `oscap` (`openscap-scanner` - the container has to be able to scan itself)
-- You may want to include another packages, as base images tend to be bare-bone and tests may require more packages to be present.
-
-#### Using Podman
-
-NOTE: With Podman, you have to run all the operations as root. Podman supports rootless containers, but the test suite internally uses a container exposing a TCP port. As of Podman version 0.12.1.2, port bindings are not yet supported by rootless containers.
-
-The root user therefore needs an key without passphrase, so it can log in to the container without any additional interaction.
-Root user typically doesn't have a SSH key, or it has one without a passphrase.
-Use this test to find out whether it has a key:
-
+1. write a pass script into *DIR* - (some rules can have more than one pass scenario)
+1. run `test_suite.py` with command:
 ```
-sudo test -f /root/.ssh/id_rsa && echo "Root user already has an id_rsa key" || echo "Root user has no id_rsa key"
-```
-
-If there is no key, it is safe to create one:
-
-```
-ssh-keygen -f id_rsa -N ""
-sudo mkdir -p /root/.ssh
-sudo chmod go-rwx /root/.ssh
-sudo mv id_rsa* /root/.ssh
-```
-
-In any case, `root` now has an `id_rsa` key without passphrase, so let's build the container.
-Go into the `Dockerfiles` directory of the project, and execute the following:
-
-```
-public_key="$(sudo cat /root/.ssh/id_rsa.pub)"
-podman build --build-arg CLIENT_PUBLIC_KEY="$public_key" -t ssg_test_suite -f test_suite-rhel .
-```
-
-#### Using Docker
-
-The procedure is same as using Podman, you just swap the `podman` call with `docker`, which moreover needs root privileges.
-
-### Running the tests
-
-This is an example to run test scenarios for rule `rule_sshd_disable_kerb_auth`.
-Note that you need to have root privileges for both cases:
-
-Using Podman:
-
-```
-./test_suite.py rule --container ssg_test_suite --datastream ../build/ssg-centos7-ds.xml --xccdf-id scap_org.open-scap_cref_ssg-rhel7-xccdf-1.2.xml rule_sshd_disable_kerb_auth
-```
-
-Using Docker:
-
-```
-./test_suite.py rule --docker ssg_test_suite --datastream ../build/ssg-centos7-ds.xml --xccdf-id scap_org.open-scap_cref_ssg-rhel7-xccdf-1.2.xml rule_sshd_disable_kerb_auth
-```
-
-Also, as containers may get any IP address, a conflict may appear in your local client's `known_hosts` file.
-You might have a version of `oscap-ssh` that doesn't support ssh connection customization at the client-side, so it may be a good idea to disable known hosts checks for all hosts if you are testing on a VM or under a separate user.
-You can do that by putting following lines in your `$HOME/.ssh/config` file:
-
-```
-StrictHostKeyChecking no
-UserKnownHostsFile /dev/null
+./test_suite.py rule --libvirt qemu:///session ssg-test-suite-fedora --datastream ../build/ssg-fedora-ds.xml accounts_password_minlen_login_defs
 ```
+Example of test scenarios for this rule can be found at: #3697
```

Comment:
Can we use a link here?
`[#3697](https://github.com/ComplianceAsCode/content/pull/3697)`